agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -1209,9 +1209,9 @@ func (dg *dockerGoClient) removeVolume(ctx context.Context, name string) error {
 		return &CannotGetDockerClientError{version: dg.version, err: err}
 	}
 
-	ok := client.VolumeRemove(ctx, name, false)
-	if ok != nil {
-		return &CannotRemoveVolumeError{err}
+	removeErr := client.VolumeRemove(ctx, name, false)
```

Comment:
nit: this can just `err`

Reply:
i wanted to make it very distinctively separate from the err above this, but I don't feel strongly about it, I'll update it to use err for consistency.